easybuild/easyconfigs/s/SeqAn/SeqAn-2.4.0-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,36 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - https://hprc.tamu.edu
+
+easyblock = 'CMakeMake'
+
+name = 'SeqAn'
+version = '2.4.0'
+
+homepage = 'https://github.com/seqan/seqan'
+
+description = """
+ SeqAn is an open source C++ library of efficient algorithms and data structures
+ for the analysis of sequences with the focus on biological data.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = ['https://github.com/seqan/seqan/archive/']
+sources = ['%(namelower)s-v%(version)s.tar.gz']
+checksums = ['d7084d17729214003e84818e0280a16f223c8f1c6a30eeef040c27e0c0047bd7']
+
+builddependencies = [
+    ('binutils', '2.34'),
+    ('CMake', '3.16.4'),
+]
+
+dependencies = [
+    ('bzip2', '1.0.8'),
+    ('zlib', '1.2.11'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/seqcons2', 'share/man/man1/sak.1'],
+    'dirs': []
```

Comment:
@smoors @JackPerdue Existing `SeqAn` easyconfigs also copy the `include` dir + `'util/cmake/seqan-config.cmake'`, so should be done here too..

Reply:
I do not know why I did not see pre-existing SeqAn build.   Apologies to @pescobar for claiming authorship.   I will resubmit with those postinstallcmds and restore authorship to Pablo.

Reply:
@JackPerdue Would be easier if @smoors can fix this PR...

Reply:
OK... all I ask is that my "Author" line be removed and Pablo's restored.